### PR TITLE
Split deployment configuration

### PR DESCRIPTION
**Pull request by rsevilla87**
### Description

Split cluster deployment configuration into data-plane and control-plane.
As part of this PR, also:
- Reducing the control-plane sizing in data-plane DAGs. Going from 16x128 to 4x32
- Reducing infrastructure nodes sizing in small-scale control-plane DAGs (27 nodes). Going from 48x192 to
8x32

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

**Review by jtaleric — COMMENTED** [review]
lgtm

one question : the workload node is still pretty large, is this just due to the router test?

**Comment by rsevilla87** [comment]
> lgtm
> 
> one question : the workload node is still pretty large, is this just due to the router test?

Good catch, we can reduce it's size in control-plane (it's only used to host the kube-burner pod) and keep the original size for data-plane

**Review by jtaleric — APPROVED** [review]
lgtm

File: dags/openshift_nightlies/config/install/aws/ovn-large-cp.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "openshift_worker_root_volume_iops": 0,
     "machineset_metadata_label_prefix": "machine.openshift.io",
     "openshift_infra_node_instance_type": "m5.12xlarge",
-    "openshift_workload_node_instance_type": "m5.4xlarge",
+    "openshift_workload_node_instance_type": "m5.2xlarge",
```
Comment (mohit-sheth) [inline dags/openshift_nightlies/config/install/aws/ovn-large-cp.json:21]: m5.2xlarge is good enough for benchmark-operator pods and kube-burner/MB pod?

Comment (rsevilla87) [reply dags/openshift_nightlies/config/install/aws/ovn-large-cp.json:21]: This is the control-plane dag config, no MB here. kube-burner would have enough resources.

File: dags/openshift_nightlies/config/install/alibaba/ovn-large-cp.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "openshift_worker_root_volume_iops": 0,
     "machineset_metadata_label_prefix": "machine.openshift.io",
     "openshift_infra_node_instance_type": "ecs.g6.8xlarge",
-    "openshift_workload_node_instance_type": "ecs.g6.4xlarge",
+    "openshift_workload_node_instance_type": "ecs.g6.8xlarge",
```
Comment (afcollins) [inline dags/openshift_nightlies/config/install/alibaba/ovn-large-cp.json:24]: 32cpu x 132 GB required for workload node? Larger than others that are 8x32.

**Comment by afcollins** [comment]
missing aws-arm configs. I think dags would not render without them

File: dags/openshift_nightlies/config/install/alibaba/ovn-large-cp.json (hunk 2 above)
Comment (rsevilla87) [reply dags/openshift_nightlies/config/install/alibaba/ovn-large-cp.json]: Good catch, fixed!

File: dags/openshift_nightlies/config/install/aws-arm/ovn-cp.json [hunk 3]
```diff
@@ -7,26 +7,26 @@
     "openshift_worker_count": 3,
     "openshift_host_prefix": "22",
     "openshift_network_type": "OVNKubernetes",
-    "openshift_master_instance_type": "m6g.4xlarge",
+    "openshift_master_instance_type": "r6g.4xlarge",
```
Comment (afcollins) [inline dags/openshift_nightlies/config/install/aws-arm/ovn-cp.json:10]: I had lot of problems creating clusters with r6g that went away with m6g.

Comment (rsevilla87) [reply dags/openshift_nightlies/config/install/aws-arm/ovn-cp.json]: Ok, rolling it back to m6g

File: dags/openshift_nightlies/config/install/aws-arm/sdn-cp.json [hunk 4]
```diff
@@ -6,27 +6,27 @@
     "openshift_master_count": 3,
     "openshift_worker_count": 3,
     "openshift_host_prefix": "22",
-    "openshift_network_type": "OpenShiftSDN",
-    "openshift_master_instance_type": "m6g.4xlarge",
+    "openshift_network_type": "OpenshiftSDN",
```
[the remark this answers is not in the document]
Comment (rsevilla87) [reply dags/openshift_nightlies/config/install/aws-arm/sdn-cp.json]: fixed!

**Review by afcollins — APPROVED** [review]
LGTM!
let's merge this so I can update my WIP branches lol

**Pull request merged**